onnxruntime/core/graph/graph.cc

```diff
@@ -762,7 +763,7 @@ Status Node::LoadFromOrtFormat(const onnxruntime::fbs::Node& fbs_node, const log
         subgraphs_.push_back(std::move(subgraph));
       }
 
-      AddAttribute(attr_proto.name(), std::move(attr_proto));
+      AddAttributeProto(std::move(attr_proto));
```

Comment:
this actually fixes a bug since `attr_proto` could be moved from before `attr_proto.name()` is called. could put this in a smaller PR if we want it for the release. in practice, I observed that the name in the NodeAttributes map was empty in some unit tests.